hack/tools.go

```diff
@@ -22,6 +22,7 @@ package tools
 // This package imports things required by this repository, to force `go mod` to see them as dependencies
 import (
 	_ "github.com/coreydaley/openshift-goimports"
+	_ "github.com/polyfloyd/go-errorlint"
```

Comment:
Mind using the go-install pattern? https://github.com/kcp-dev/kcp/blob/b89a696c843e8da348c810af94cb2b55d961480c/Makefile

Reply:
Should probably file an issue to do this for openshift-goimports too

Reply:
Filed #256 for openshift-goimports & I'll make that change